### PR TITLE
Fixes for Base_Scripts, adding scrapers

### PR DESCRIPTION
**Pull request by CaptainStabs**

### CODE REVIEW

File: Base_Scripts/Scrapers/list_pdf_extractor/list_pdf_extractor.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import requests
+import os
+from bs4 import BeautifulSoup
+import urllib
+import re
+import time
+
+__noted__ = 'fixes shamelessly stolen from dunnousername without credit'  # Just don't delete this
```
Comment (mcsaucy) [inline Base_Scripts/Scrapers/list_pdf_extractor/list_pdf_extractor.py:10]: Why is this needed?

File: Base_Scripts/Scrapers/list_pdf_extractor/list_pdf.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+If this does not make sense, try checking the comments within the code.
+ Working example can be found [here](https://github.com/CaptainStabs/Scrapers/blob/master/USA/CA/alameda/alameda_scraper.py)
+
+# Versions:
+`list_pdf_extractor.py` : most basic of the scripts, mostly used for reference
+
+`list_pdf_extractor_v2.py` : Uses imported `get_files` function. Useful for cases where a custom `get_files` is **not** needed. Function can be found [here](https://github.com/CaptainStabs/Scrapers/blob/master/common/bs_scrapers/get_files.py)
+
+`list_pdf_extractor_v3.py` : Built off of V2, Allows for filtering links by common unwanted words. See [golden_west_scraper.py](https://github.com/CaptainStabs/Scrapers/blob/master/USA/CA/golden_west_college/golden_west_scraper.py) for working example.
```
Comment (mcsaucy) [inline Base_Scripts/Scrapers/list_pdf_extractor/list_pdf.md:36]: Would it be possible to refactor this into a single library? Would probably make consumption easier, without relying upon copy-pasting code all over the place. While this may be fine for a few scrapers, I can see a future where we need to make a broad change and have to update dozens or hundreds of very similar but slightly different scrapers.

File: Base_Scripts/Scrapers/list_pdf_extractor/list_pdf.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+This script has two functions, the first, `extract_info`, extracts the links containing documents, and saves the url and the document name to a file called `links.txt`
+The second function, `get_files`, reads the link and name from `links.txt` and downloads the files.  
+
+
+# More in depth explanations (Poorly explained, nerdy stuff)
+ `extract_info` uses `urllib` to open the webpage, and then `BeautifulSoup4` to parse it. It then uses regex to find all links that end with pdf or doc. It needs a few lines to be replaced with regex.
```
Comment (mcsaucy) [inline Base_Scripts/Scrapers/list_pdf_extractor/list_pdf.md:44]: IMO these would be better suited as comments in a library that other scrapers can consume.

File: USA/CA/beaumont/DEPRECATD_beaumont_stat_scraper.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
+import requests
```
Comment (mcsaucy) [inline USA/CA/beaumont/DEPRECATD_beaumont_stat_scraper.py:1]: Why is this file deprecated? If it's not needed anymore, should we just remove it?

File: Base_Scripts/Scrapers/list_pdf_extractor/list_pdf.md (hunk 2 above)
Comment (mcsaucy) [reply Base_Scripts/Scrapers/list_pdf_extractor/list_pdf.md:36]: Even throughout the rest of this PR I'm seeing a ton of reuse and it's already hard to review. Having a expansive library (which we diligently review) and does most of the heavy lifting could make individual scraper reviews (of which there will be *many*) much easier.

**Review by mcsaucy — CHANGES_REQUESTED** [review]
As a general note, this PR is gigantic and seems to be changing a lot. Small PRs are way easier to review, so you'll probably get faster reviews (from actual project members; I can't merge).

File: USA/CA/beaumont/DEPRECATD_beaumont_stat_scraper.py (hunk 4 above)
Comment (CaptainStabs) [reply USA/CA/beaumont/DEPRECATD_beaumont_stat_scraper.py]: Removed

File: Base_Scripts/Scrapers/list_pdf_extractor/list_pdf.md (hunk 2 above)
Comment (CaptainStabs) [reply Base_Scripts/Scrapers/list_pdf_extractor/list_pdf.md:43]: Will work on in next PR

File: Base_Scripts/Scrapers/list_pdf_extractor/list_pdf_extractor.py (hunk 1 above)
Comment (CaptainStabs) [reply Base_Scripts/Scrapers/list_pdf_extractor/list_pdf_extractor.py]: It's not.

**Comment by CaptainStabs** [comment]
As it will only get bigger, going to go ahead and merge it.

**Comment by mcsaucy** [comment]
> As it will only get bigger, going to go ahead and merge it.
> 
> 

A way to mitigate this going forward may be feature branches, but that may require significant changes to your workflow
